contracts/AnchoredView/AnchoredView.sol

```diff
@@ -3,30 +3,15 @@ pragma experimental ABIEncoderV2;
 
 import "./SymbolConfiguration.sol";
 import "../OpenOraclePriceData.sol";
-
-interface AnchorOracle {
-    function numBlocksPerPeriod() external view returns (uint); // approximately 1 hour: 60 seconds/minute * 60 minutes/hour * 1 block/15 seconds
-
-    function assetPrices(address asset) external view returns (uint);
-
-    struct Anchor {
-        // floor(block.number / numBlocksPerPeriod) + 1
-        uint period;
-
-        // Price in ETH, scaled by 10**18
-        uint priceMantissa;
-    }
-    function anchors(address asset) external view returns (Anchor memory);
-}
-
+import "../OpenOracleData.sol";
 
 /**
  * @notice Price feed conforming to Price Oracle Proxy interface.
  * @dev Use a single open oracle reporter and anchored to and falling back to the Compound v2 oracle system.
  * @dev The reporter must report at a minimum the USD/ETH price, so that anchor ETH/TOKEN prices can be converted to USD/TOKEN
  * @author Compound Labs, Inc.
  */
-contract AnchoredView is SymbolConfiguration {
+abstract contract AnchoredView is SymbolConfiguration, OpenOracleData {
```

Comment:
This is for `source`? It's not really a data contract...

Reply:
yea just for source... should we rename OOData? or just copy paste the function into this file?